### PR TITLE
Fix StoreTest for internal CI.

### PR DESCRIPTION
**Pull request by jasonwyatt**
Related Bug:

http://b/145369386

Plan to resolve this is to merge this branch (I've tested the code changes here by manually copying it to the internal repo and running the test there), then to update copybara's exclusion list to remove this test.

### CODE REVIEW

File: javatests/arcs/core/storage/StoreTest.kt [hunk 1]
```diff
@@ -343,39 +315,61 @@ class StoreTest {
...
-        DriverFactory.register(provider)
+    private fun setupFakes(): Pair<FakeDriver<CrdtCount.Data>, FakeProvider> {
+        val fakeDriver = FakeDriver<CrdtCount.Data>()
+        val fakeProvider = FakeProvider(fakeDriver)
+        DriverFactory.register(fakeProvider)
+        return fakeDriver to fakeProvider
+    }
 
-        return driver to provider
+    private fun setupSetMocks(): Pair<FakeDriver<CrdtSet.Data<*>>, FakeProvider> {
```
Comment (galganif) [inline javatests/arcs/core/storage/StoreTest.kt:328]: should this be 'setupSetFakes' ?

File: javatests/arcs/core/storage/StoreTest.kt [hunk 2]
```diff
@@ -261,20 +240,18 @@ class StoreTest {
 
         // Note that this assumes no asynchrony inside store.ts. This is guarded by the following
```
Comment (galganif) [inline javatests/arcs/core/storage/StoreTest.kt:241]: store.ts?

File: javatests/arcs/core/storage/StoreTest.kt (hunk 1 above)
Comment (jasonwyatt) [reply javatests/arcs/core/storage/StoreTest.kt:328]: Ah yes, thanks.

File: javatests/arcs/core/storage/StoreTest.kt (hunk 2 above)
Comment (jasonwyatt) [reply javatests/arcs/core/storage/StoreTest.kt:241]: lol 6-month-old copypaste typo :D

**Pull request merged**